.github/workflows/build-all-site.yml

```diff
@@ -0,0 +1,42 @@
+name: PR Build documentation site
```

Comment:
can we  rename the file into build-pr-site.yml? This will explicit that it is for PR only. In https://github.com/bonitasoft/bonita-doc/pull/2134/files, we name the file _build-pr-preview.yml_
If so, rename the 'paths' config as well